### ISSUE TITLE
Move rules_svelte to Bazel org

### ISSUE THREAD

**Issue by thelgevold** [bazelbuild/bazel#7685]
### Description of the problem / feature request:

I am creating this issue to see if there is interest in moving my experimental Svelte rules to the Bazel org for better visibility. Please help me identify the requirements for transferring the repo.

Here is my current repo: https://github.com/thelgevold/rules_svelte

There is also a seed project using the rules here: https://github.com/thelgevold/svelte-bazel-example

Thanks,
Tor

**Comment by aiuto** [comment]
We're working on guidance for moving things to bazelbuild.

We would like to see new repos under the org be part of the CI system and part of the Federation.
Since we are still ramping that effort up, it may be a while before we can accept the new one.

In the meantime, we do know a few prerequisite things must happen.
- add unit tests 
- add tests for interoperability with rules_nodejs and rules_sass
- Run "buildifier --warn=fix"
- Make sure your code works with all incompatible changes (try bazelisk --migrate)
- work with the EngProd team to have us add your repo to our CI
   https://github.com/bazelbuild/continuous-integration/issues/new/choose
- establish who would help own the code. we want to prevent orphans.

_Note: the above is mostlyis a repeat of the mail thread, 
https://groups.google.com/d/msgid/bazel-dev/CACpMnuseB1qZLfGhxuQFBPYJD0r8vH13ZYJE%3DgqRQzD6bwMfSA%40mail.gmail.com
Having it in an issue makes it easier to find and track._

**Comment by thelgevold** [comment]
Hi.
Thanks for this list! 

I will take a look an report back later.

**Comment by meisterT** [comment]
What's the status of this?

**Comment by thelgevold** [comment]
Hi. Sorry, but I haven't been able to make much progress towards this.
Maybe it's best to put it on hold and revisit later?

**Comment by sgowroji** [comment]
Hi there! We're doing a clean up of old issues and will be closing this one. Please reopen if you’d like to discuss anything further. We’ll respond as soon as we have the bandwidth/resources to do so.
